Overview.src.html

```diff
@@ -3210,6 +3323,73 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <li>If <var>cookies</var> is not the empty string, append
      `<code title=http-cookies>Cookie</code>`/<var>cookies</var> to <var>httpRequest</var>'s
      <span title=concept-request-header-list>header list</span>.
+
+     <li><p>If the user agent supports <span title=concept-token-binding>Token Binding</span>,
+       build a <code title="">Sec-Token-Binding</code> <span title=concept-header>header</span>
+       by running these substeps:
+       <p class="note no-backref">A <code title="">Sec-Token-Binding</code>
+       <span title=concept-header>header</span> allows a server
+       to bind cookies that it issues to a <span title=concept-token-binding-id>Token Binding Id</span>,
+       and to verify that cookies presented by the user agent are bound to the correct
+       <span title=concept-token-binding-id>Token Binding Id</span>.
+
+       <ol>
+         <li><p>Set the <var>httpRequest</var>'s
+           <span title=concept-request-use-token-binding>use-token-binding flag</span>.
```

Comment:
Isn't this a property of the connection, not the request? It seems so, and also seems like you'd be able to replace most of the "If the user agent supports token binding..." bits with "If |connection|'s <a>token binding metadata</a> is not `null`..."